shared/fs/really-delete/index.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import * as React from 'react'
+import * as Constants from '../../constants/fs'
+import {Box, ConfirmModal, HeaderOnMobile, Icon, MaybePopup, ProgressIndicator} from '../../common-adapters'
+import {globalStyles, globalMargins} from '../../styles'
+import * as Types from '../../constants/types/fs'
+
+export type Props = {
+  onBack: () => void,
+  onDelete: () => void,
+  path: Types.Path,
+  title: string,
+}
+
+const _Spinner = (props: Props) => (
+  <MaybePopup onClose={props.onBack}>
+    <Box
+      style={{...globalStyles.flexBoxColumn, alignItems: 'center', flex: 1, padding: globalMargins.xlarge}}
+    >
+      <ProgressIndicator style={{width: globalMargins.medium}} />
+    </Box>
+  </MaybePopup>
+)
+const Spinner = HeaderOnMobile(_Spinner)
+
+const Header = (props: Props) => <Icon type="iconfont-trash" sizeType="Big" />
+
+const _ReallyDeleteFile = (props: Props) =>
+  !!props.path && (
+    <ConfirmModal
+      confirmText={`Yes, delete it.`}
```

Comment:
`confirmText="Yes, delete it."`, same below.

Reply:
This seems more readable to me than escaping the quotes and manually concatenating the path in.

Reply:
I think @songgao is saying that you can leave everything the same, except not escape to JS to put the string in.
```
<ConfirmModal confirmText="Yes, delete it." />
```

Reply:
er, I changed this one. I'm referring to the below. e.g. ```prompt={`Are you sure you want to delete "${Types.pathToString(this.props.path)}"?`}```